libs/clients/fiskistofa/src/lib/api.ts

```diff
@@ -0,0 +1,219 @@
+import { AuthHeaderMiddleware } from '@island.is/auth-nest-tools'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import { Inject, Injectable } from '@nestjs/common'
+import { ConfigType } from '@nestjs/config'
+import {
+  Configuration,
+  SkipApi,
+  StadaSkipsApi,
+  V1SkipHeitiHeitiGetRequest,
+  V1SkipSkipnumerGetRequest,
+  V1StadaskipsKvotategundirAlmanaksarArGetRequest,
+  V1StadaskipsKvotategundirFiskveidiarFiskveidiarGetRequest,
+  V1StadaskipsSkipnumerAlmanaksarArDeilistofnarBreyttPostRequest,
+  V1StadaskipsSkipnumerAlmanaksarArDeilistofnarGetRequest,
+  V1StadaskipsSkipnumerFiskveidiarFiskveidiarBreyttPostRequest,
+  V1StadaskipsSkipnumerFiskveidiarFiskveidiarGetRequest,
+  V1StadaskipsSkipnumerFiskveidiarFiskveidiarKvotiBreyttPostRequest,
+} from '../../gen/fetch'
+import { FiskistofaClientConfig } from './fiskistofaClient.config'
+import {
+  mapShipStatusForTimePeriod,
+  mapShipStatusForCalendarYear,
+  mapQuotaType,
+} from './utils'
+import { FetchError } from '@island.is/clients/middlewares'
+
+@Injectable()
```

Comment:
By doing the following, other devs can start the API without having all of the required fiskistofa secrets. It'll just crash the queries which depend on fiskistofa. But in production it will stop the release if the secrets are missing.

```suggestion
import { LazyDuringDevScope } from '@island.is/nest/config'

@Injectable({ scope: LazyDuringDevScope })
```

Reply:
Ok, I had no idea that existed. Thanks for letting me know :)